baseband/vlbi_base/base.py

```diff
@@ -189,18 +190,18 @@ def tell(self, unit=None):
             return self.start_time + self.tell(unit=u.s)
 
         return (self.offset * u_sample).to(unit, equivalencies=[(u.s, u.Unit(
-            self.samples_per_frame * self.frames_per_second * u_sample))])
+            self._sample_rate * u_sample))])
```

Comment:
Use `self.sample_rate.to_value(u.Hz) * u_sample`.  Though I actually forget why one couldn't just do `(self.offset / self.sample_rate).to(unit)`. Can you try that?

Reply:
The latter does work!